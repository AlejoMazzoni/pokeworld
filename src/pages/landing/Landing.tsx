import styles from "./Landing.module.scss";

const Landing = () => {
  return (
    <div className={styles.landingContainer}>
      <div className={styles.infoContainer}>
        <h1>PokeWorld</h1>
        <p>This is an app made by Alejo Mazzoni for practice purposes</p>
      </div>
    </div>
  );
};

export default Landing;
