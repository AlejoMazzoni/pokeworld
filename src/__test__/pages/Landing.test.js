import renderer from "react-test-renderer";
import Landing from "../../pages/landing/Landing";

describe("Landing", () => {
  it("renders everything correctly", () => {
    const component = renderer.create(<Landing />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
