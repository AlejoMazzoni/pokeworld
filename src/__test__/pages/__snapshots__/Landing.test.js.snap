// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing renders everything correctly 1`] = `
<div
  className="landingContainer"
>
  <div
    className="infoContainer"
  >
    <h1>
      PokeWorld
    </h1>
    <p>
      This is an app made by Alejo Mazzoni for practice purposes
    </p>
  </div>
</div>
`;
